tests/integration/api/test-crud-functions.php

```diff
@@ -321,7 +336,6 @@ protected function get_relationship_ids( int $post_id ) {
 		return array_map( 'intval', wp_list_pluck( $result, 'id2' ) );
 	}
 
-
 	/**
 	 * Get an array of valid data for testing insert_entry_model().
```

Comment:
Same, but pre-existing.

```suggestion
	 * Get an array of valid data for testing insert_model_entry().
```